--- pth-2.0.0/pth.m4.orig	2003-12-18 20:48:52.000000000 +0100
+++ pth-2.0.0/pth.m4	2003-12-18 20:49:27.940529008 +0100
@@ -40,7 +40,7 @@
 dnl ##
 dnl
 dnl #   auxilliary macros
-AC_DEFUN(_AC_PTH_ERROR, [dnl
+AC_DEFUN([_AC_PTH_ERROR], [dnl
 AC_MSG_RESULT([*FAILED*])
 define([_ac_pth_line],[dnl
 "+------------------------------------------------------------------------+"])
@@ -52,13 +52,13 @@
 undefine([_ac_pth_line])
 exit 1
 ])
-AC_DEFUN(_AC_PTH_VERBOSE, [dnl
+AC_DEFUN([_AC_PTH_VERBOSE], [dnl
 if test ".$verbose" = .yes; then
     AC_MSG_RESULT([  $1])
 fi
 ])
 dnl #   the user macro
-AC_DEFUN(AC_CHECK_PTH, [dnl
+AC_DEFUN([AC_CHECK_PTH], [dnl
 dnl
 dnl #   prerequisites
 AC_REQUIRE([AC_PROG_CC])dnl
