--- pth.m4~	Sat Mar 24 15:49:50 2001
+++ pth.m4	Sun Aug 11 20:45:02 2002
@@ -42,14 +42,14 @@
 dnl #   auxilliary macros
 AC_DEFUN(_AC_PTH_ERROR, [dnl
 AC_MSG_RESULT([*FAILED*])
-define(_ac_pth_line,dnl
-"+------------------------------------------------------------------------+")
+define([_ac_pth_line],[dnl
+"+------------------------------------------------------------------------+"])
 echo " _ac_pth_line" 1>&2
 cat <<EOT | sed -e 's/^[[ 	]]*/ | /' -e 's/>>/  /' 1>&2
 $1
 EOT
 echo " _ac_pth_line" 1>&2
-undefine(_ac_pth_line)
+undefine([_ac_pth_line])
 exit 1
 ])
 AC_DEFUN(_AC_PTH_VERBOSE, [dnl
